### PR TITLE
Completed TODO in file

### PR DESCRIPTION
**Pull request by kyscg**
Added various scenarios as asked in the TODO

### CODE REVIEW

**Comment by kyscg** [comment]
Review pleasee?? @rthadur @angersson

File: tensorflow/lite/kernels/concatenation_test.cc [hunk 1]
```diff
@@ -26,9 +26,24 @@ using ::testing::ElementsAreArray;
 
 class BaseConcatenationOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, axis, input
-  // dimensions.
   BaseConcatenationOpModel() {}
+  BaseConcatenationOpModel(
```
Comment (jianlijianli) [inline tensorflow/lite/kernels/concatenation_test.cc]: It seems the new "BaseConcatenationOpModel" class is very similar to existing one with the same name. Could you please merge the two?

Something like this:
BaseConcatenationOpModel(const TensorData& input_template, int axis,
                           int num_inputs, const ActivationFunctionType& fused_activation_function = ActivationFunctionType_NONE) {
.....
}

**Review by jianlijianli — APPROVED** [review]
Thanks for working on this. The logic looks good. Just a comment on better organizing the code. Thanks.

File: tensorflow/lite/kernels/concatenation_test.cc (hunk 1 above)
Comment (rthadur) [reply tensorflow/lite/kernels/concatenation_test.cc]: @kyscg gentle ping

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**